src/types.rs

```diff
@@ -85,3 +85,9 @@ pub enum MetricType {
 }
 
 pub type Bytes = Vec<u8>;
+
+pub enum ContextType {
```

Comment:
Please add:
```
#[repr(u32)]
#[derive(Debug)]
```
(it's not used in the ABI right now, but it's going to be soon, so we might as well define it properly now)

Also, could you move it below `Status`?

Reply:
done